docs/rules/forbid-component-props.md

```diff
@@ -50,8 +50,18 @@ custom message, and a component whitelist:
 ```js
 {
   "propName": "someProp",
-  "allowedFor": [SomeComponent, AnotherComponent],
-  "message": "Avoid using someProp"
+  "allowedFor": ["SomeComponent", "AnotherComponent"],
+  "message": "Avoid using someProp except SomeComponent and AnotherComponent"
+}
+```
+
+Use `disallowedFor` as a blacklist to lint props for specific components. `disallowedFor` should have at least one item.
```

Comment:
Let's use inclusive language and keep the terminology that this rule uses: replace "blacklist" with "disallowlist" or "blocklist"

(I'm leaving this as a comment and not a "request changes" because I'm not sure if the community members are allowed to request changes to others' PRs.) Totally up to @ljharb whether you require this too or not.